cmd/api/api.go

```diff
@@ -94,11 +109,11 @@ func shutdown() {
 	defer Conf.API.DB.Close()
 }
 
-func readinessResponse(w http.ResponseWriter, r *http.Request) {
-	statusCocde := http.StatusOK
+func readinessResponse(c *gin.Context) {
+	statusCode := http.StatusOK
 
 	if Conf.API.MQ.Connection.IsClosed() {
-		statusCocde = http.StatusServiceUnavailable
+		statusCode = http.StatusServiceUnavailable
```

Comment:
not sure i follow why we need MQ checks here,the API does not do anything with the broker.